data/json/hobbies.json

```diff
@@ -91,5 +91,114 @@
     "description": "You enjoyed camping in the vast wilderness as a means of escaping the pace of modern life.  The wilderness isn't filled with people, so it must be safe.  Right?  It must be.",
     "points": 2,
     "skills": [ { "level": 2, "name": "survival" }, { "level": 2, "name": "firstaid" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "socialite",
+    "name": "High Society Social",
+    "description": "You attended some very impressive galas, made some great speeches, and you were always a great host.",
+    "points": 3,
+    "proficiencies": [ "prof_winemaking", "prof_cheesemaking_1" ],
+    "traits": [ "LIAR", "STYLISH" ],
+    "skills": [ { "level": 3, "name": "speech" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "engineering",
+    "name": "Engineering",
+    "description": "You studied as a generalist engineer, and you love to tinker.  Those skills sure are coming in handy during the apocalypse.",
+    "points": 3,
+    "skills": [
+      { "level": 2, "name": "computer" },
+      { "level": 2, "name": "mechanics" },
+      { "level": 2, "name": "chemistry" },
+      { "level": 1, "name": "bashing" }
+    ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "accounting",
+    "name": "Accounting",
+    "description": "You were always good with numbers and you filed your taxes on time each year.  When you were 11, you audited your parents, and they grounded you.",
+    "points": 2,
+    "traits": [ "PACKMULE" ],
```

Comment:
Why packmule?

Reply:
You're optimizing the space. It's a form of math problem.